### PR TITLE
README: added installation instructions for Fedora

### PR DESCRIPTION
**Pull request by ardevd**
Added installation instructions for Fedora. Tested and confirm to work on Fedora 31.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,13 @@ sudo apt-get install libxcb-render0-dev libffi-dev python-dev python-cffi
 pip install flashfocus
 ```
 
+### Fedora
+
+```
+dnf install libffi-devel python-devel python-cffi
+sudo pip install flashfocus
```
Comment (fennerm) [inline README.md:37]: Could you remove the sudo in this line? prefer not to be opinionated about where people are installing the package.

**Comment by ardevd** [comment]
I'll take care of it :)

**Comment by ardevd** [comment]
Done.

**Comment by fennerm** [comment]
Really sorry about how long this sat in review! Merged now

**Pull request merged**